packages/web/src/pages/react-query/components/TestCollectionUpdate.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react'
+
+import { Button, Flex, Text, TextInput } from '@audius/harmony'
+
+import { useCollection } from 'hooks/useCollection'
+import { useUpdateCollection } from 'hooks/useUpdateCollection'
+
+type Props = {
+  playlistId: string
+}
+
+export const TestCollectionUpdate = ({ playlistId }: Props) => {
+  const { data: collection } = useCollection(playlistId)
+  const updateCollection = useUpdateCollection()
+  const [newTitle, setNewTitle] = useState('')
+
+  const handleUpdateTitle = () => {
+    if (!collection) return
+    updateCollection.mutate({
+      playlistId,
+      metadata: {
```

Comment:
if this `metadata` arg is meant to be a sparse update, maybe we should call it `update` or `partialMetadata` something

Reply:
oh yes smarttt